WordPress/Classes/ViewRelated/Post/PostEditor.swift

```diff
@@ -111,6 +111,9 @@ protocol PostEditor: class, UIViewControllerTransitioningDelegate {
     var autosaver: Autosaver { get set }
     /// true if the post is the result of a reblog
     var postIsReblogged: Bool { get set }
+
+    /// Returns the word counts of the content in the editor.
+    var wordCount: UInt? { get }
```

Comment:
Would it make sense to make this non-optional?

I'm not sure how to interpret the difference between 0 and nil. It seems more like this is needed because of how we count words in Gutenberg. Maybe it makes more sense to default to 0 in the gutenberg side implementation instead?

Reply:
The idea was that nil is used when we don't know the count yet. When that happens we are sending zero anyway in stats so I agree with your suggestion and just return zero on that cases.